frontend/src/components/common/content/HeadingBlock.tsx

```diff
@@ -0,0 +1,15 @@
+import { Heading } from "@chakra-ui/react";
+import React from "react";
+
+import {
+  ContentBlockProps,
+  HeadingBlockState,
+} from "../../../types/ContentBlockTypes";
+
+const HeadingBlock = ({
+  block: { content },
+}: ContentBlockProps<HeadingBlockState>): React.ReactElement => {
+  return <Heading fontSize={content.size}>{content.text}</Heading>;
```

Comment:
For `a11y` purposes, it might be good to use the appropriate heading elements as well (`as="h1"`, etc.).